projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.ts

```diff
@@ -62,23 +61,9 @@ export class ExitCallsTableCellRendererComponent extends TableCellRendererBase<E
     @Inject(TABLE_ROW_DATA) rowData: Trace
   ) {
     super(columnConfig, index, parser, cellData, rowData);
-    this.apiCalleeNameCount = this.getMaxShowAPICalleeNameCount(rowData.apiCalleeNameCount);
+    const apiCalleeNameCount: string[][] = Object.entries(Object(rowData.apiCalleeNameCount));
```

Comment:
nit: Are these extra constructors (`Object()` here and `Number()` below) necessary? Or is the data already in those types?

Reply:
In this data is coming as a type called **unknown**
So I had to type cast these in appropriate values. Or is there any better way to handle this type called **unknown**?

Reply:
This would be fixed by the other comment thread below - the reason we don't have types is that the rest of the row isn't known (another indication we shouldn't be using it). We can and should declare the type of the data we expect so - instead of `TableCellRendererBase<ExploreValue, Trace>` it would be, for example, `TableCellRendererBase<Dictionary<number>>` (if we expect to receive an object like `{ "some-service": 10 }`)

This would change the type on the incoming data to `@Inject(TABLE_CELL_DATA) cellData: Dictionary<number>`